images/gitlab-error-processor/app.py

```diff
@@ -32,30 +32,31 @@ async def gitlab_webhook_consumer(request: Request):
     if job_input_data["build_status"] != "failed":
         return Response("Not a failed job, no action needed.", status_code=200)
 
-    with open(Path(__file__).parent / "job-template.yaml") as f:
-        job_template = yaml.safe_load(f)
-
-    for container in job_template["spec"]["template"]["spec"]["containers"]:
-        container.setdefault("env", []).extend(
-            [dict(name="JOB_INPUT_DATA", value=json.dumps(job_input_data))]
+    for template in ("job-template.yaml", "job-template-old.yaml"):
```

Comment:
I don't have a grafana dashboard ready for the new one yet, since I wanted to wait until we have some data in the database to make one. If we remove the opensearch job now, we'll lose access to the error taxonomy dashboard until I get the new one working.